### PR TITLE
Added IAU page and updated requirements.txt

### PR DESCRIPTION
**Pull request by tusharrao198**
Added IAU page.
Still need to increase aesthetics and more content

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
+Django==3.1.6
 django-ckeditor==6.0.0
 django-js-asset==1.2.2
-Pillow==7.2.0
-pytz==2020.1
-sqlparse==0.4.1
+Pillow==8.2.0
+python-decouple==3.3
+pytz==2021.1
+sqlparse==0.3.1
```
Comment (gajrajgchouhan) [inline requirements.txt:8]: Sqlparse was downgraded ?

File: src/web_main/settings.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 ALLOWED_HOSTS = ["stac.iitmandi.co.in", "127.0.0.1", "localhost"]
 
-DEBUG = False
+DEBUG = True
```
Comment (gajrajgchouhan) [inline src/web_main/settings.py:33]: Debug should be False

**Review by gajrajgchouhan — CHANGES_REQUESTED** [review]

**Comment by gajrajgchouhan** [comment]
Nice work! Will merge after adding aesthetics !

**Comment by gajrajgchouhan** [comment]
Oh and we have to IAU to navbar also na? @LalitNM

File: src/web_main/settings.py (hunk 2 above)
Comment (tusharrao198) [reply src/web_main/settings.py:33]: Ok

File: src/web_main/settings.py [hunk 3]
```diff
@@ -30,8 +30,10 @@
 
 ALLOWED_HOSTS = ["stac.iitmandi.co.in", "127.0.0.1", "localhost"]
 
-DEBUG = False
-
+try:
```
Comment (harnaman-hk) [inline src/web_main/settings.py:33]: This try-error block should be avoided, we do not use os in production. An alternative is:
```
try:
    import secret
    SECRET_KEY = secret.SECRET_KEY
    DEBUG = False
except ImportError:
    SECRET_KEY = os.environ.get("SECRET_KEY")
    DEBUG = True
```

**Comment by LalitNM** [comment]
> Oh and we have to add IAU to navbar also na? @LalitNM

Yeah @gajrajgchouhan, but we have to make sure that the navbar is not too long.

File: src/web_main/settings.py (hunk 3 above)
Comment (tusharrao198) [reply src/web_main/settings.py:33]: Ok. Will do that

**Comment by abhijeetmanhas** [comment]
@tusharrao198 can you post the Screenshot for IAU page? Code looks fine

**Comment by abhijeetmanhas** [comment]
@tusharrao198 can you post the Screenshot for IAU page? Code looks fine

**Comment by tusharrao198** [comment]
@abhijeetmanhas , it's in the early stage right now. Need to update UI.

Screenshot from 2021-04-11 03-39-25

**Review by abhijeetmanhas — APPROVED** [review]

**Review by LalitNM — APPROVED** [review]

**Review by gajrajgchouhan — APPROVED** [review]
Looks good !

**Pull request merged**